xpk.py

```diff
@@ -999,6 +1007,60 @@ def run_gke_cluster_create_command(args) -> int:
   return 0
 
 
+def create_cluster_configmap(args, system):
+  """Run the Create GKE Cluster ConfigMap request.
+
+  Args:
+    args: user provided arguments for running the command.
+    system: system characteristics.
+
+  Returns:
+    0 if successful and 1 otherwise.
+  """
+  # TODO: Update when GPU support is enabled
+  data = f'{args.tpu_type}: "{int(args.num_slices) * system.vms_per_slice}"'
+  yml_string = cluster_configmap_yaml.format(args=args,
+                                           data=data)
+  tmp = write_temporary_file(yml_string)
+  command = f'kubectl apply -f {str(tmp.file.name)}'
```

Comment:
What happens if the cluster has a config map already with the same name? Can it apply it or does it error? We would want to be to update config maps on new calls of cluster create.

Reply:
If the ConfigMap already exist with the same map, it will update the data in the map.